### ISSUE TITLE
Issues Utilizing SearchGUI Search with Converted Files

### ISSUE THREAD

**Issue by EthanHasenoehrl** [elnurgar/mzxml-precursor-corrector#3]
Hello all,

I am utilizing data from a Waters machine and utilized this script to fix the drifting problem as described in this post. To convert the .raw Waters files to mzML files I am using a protocol [as shown here](https://ccms-ucsd.github.io/GNPSDocumentation/fileconversion/). I have tried utilizing both ProteoWizard and a custom package as described in that article before utilizing the script to adjust the Waters drifting issue. 

After converting these files and running the script to fix the drifting issue, I am using those files in the Peptide Shaker/SearchGUI programs to hopefully identify some peptides from MS/MS data. Whenever I run these files I am getting the following error: 

X! TANDEM Vengeance (2015.12.15.2)
Loading spectra (mgf). loaded.
No input spectra met the acceptance criteria.
Wed Jun 26 09:03:21 MDT 2024 X!Tandem finished for 20221017_04_1-400_modified.mgf (4.9 seconds).
Wed Jun 26 09:03:21 MDT 2024
Wed Jun 26 09:03:21 MDT 2024
Wed Jun 26 09:03:22 MDT 2024
Could not rename X!Tandem result for 20221017_04_1-400_modified.mzML.
Could not find X!Tandem result file for 20221017_04_1-400_modified.mzML.
Zipping output files.
Wed Jun 26 09:03:22 MDT 2024
Processing identification files with PeptideShaker.
2024-06-26 09:03:22.483 java[5484:232189] WARNING: Secure coding is automatically enabled for restorable state! However, not on all supported macOS versions of this application. Opt-in to secure coding explicitly by implementing NSApplicationDelegate.applicationSupportsSecureRestorableState:.
Wed Jun 26 09:03:24 MDT 2024
PeptideShaker processing canceled. Please see the PeptideShaker log file:
/xxx
Wed Jun 26 09:03:24 MDT 2024
Wed Jun 26 09:03:24 MDT 2024
Wed Jun 26 09:03:24 MDT 2024
Wed Jun 26 09:03:24 MDT 2024
Searching Canceled!
Error: Stream closed
An error occurred while running SearchGUl. Please contact the developers.
The search or processing did not finish properly!

I have utilized both X!Tandem and Comet as search engines and come across similar errors. Has anyone come across this issue before, and if so how did you address it? Any input is greatly appreciated.

Thank you.

**Comment by elnurgar** [comment]
Hi, 
I don't know the software you are using. However, I advice you to download the last version of msconvert (proteowizard) and in the main panel you keep only 1 line Waters DDA processing. That's all and you convert to mzml. I think it will work. Can you check please. If you use the last version of msconvert, no need to use my script for waters conversion.

**Comment by EthanHasenoehrl** [comment]
Hello, 

I am using version 3.0.24169-fe51169 of MS Convert. Just to clarify- the newest version of MS Convert/ Proteowizard fixes the drift issue that your script does? 

Thank you again.

**Comment by elnurgar** [comment]
I have 24.124 and it fixes the drift issue, so I think yours also. For the conversion I advice you to try with this configuration:
![image](https://github.com/elnurgar/mzxml-precursor-corrector/assets/67260554/b59218c9-9f05-4dba-b651-e108df1f376b)

**Comment by EthanHasenoehrl** [comment]
Thank you again for your reply. When utilizing the settings that you mentioned, my MS Convert says that the conversion has failed. Here's the log:

Starting...
Opening file "filepath.raw" for read.
eas pina ch mi tara Readerue reasi(Sinta milenant Ms DataLe taurs readeCoff cortef eauton byga)
at MSConvertGUI.MainLogic.process File(String filename, Config config. ReaderList readers, Map'2 usedOutput Filenames) at MSConvertGUI.Main Logic.Go(Config config, Map'2 usedOutput Filenames)

I tried with multiple files and it appeared that something wasn't working with the Waters files. I've never had issues like this with MS convert when converting Agilent files etc.

**Comment by elnurgar** [comment]
The analyses that you have realized are in DDA mode? I mean you have MS1 and MS2 spectra? What is your mass spectrometer model? We have no troubles with our data obtained from Synapt G2-Si

**Comment by EthanHasenoehrl** [comment]
Thank you for your reply,

Our mass spectrometer model is the Waters SynaptXS, and the data was acquired in MsE mode. I have attached the full configuration file if you would like any additional information about the set up.

[20221017_02_1-100.txt](https://github.com/user-attachments/files/16134837/20221017_02_1-100.txt)

Again, thank you.

**Comment by elnurgar** [comment]
Can you send me an example of data I will try to convert on my pc? You can send me to my email elnurgar@gmail.com.

**Comment by EthanHasenoehrl** [comment]
Files have been sent!

**Comment by elnurgar** [comment]
I have looked to the file conversion. There are no big issues with converting you can use this configuration:
![image](https://github.com/elnurgar/mzxml-precursor-corrector/assets/67260554/0fa20858-bf45-4b08-afa0-8642201138a1)


I looked xTandem software documentation, it accepts only DTA, PKL or MGF files. That means you should use spectra file, not chromatogram file.

**Comment by elnurgar** [comment]
You can use mzmine software if you want to export a scan to mgf, however I don't know if this mgf will be accepted by xTandem.

**Comment by EthanHasenoehrl** [comment]
Hello,

Thank you again for your response. It seems like MS Convert will process the files, but there is an issue when I attempt to read the file. I utilized a variety of different search engines to identify peptides, and was unable to read these files using the settings you outlined. It looks like X!Tandem does accept mzML files according to their [release notes](https://www.thegpm.org/tandem/release.html#:~:text=This%20mechanism%20allows%20X!,in%20collaboration%20with%20Peter%20Lobel)).

I will be attempting to use an application that I found that Waters makes called [DataConnect](https://microapps.on-demand.waters.com/home/showmarkdown/data-as-a-product) which hopefully should fix the calibration issue that your script uses. 

I'll make another post if I end up finding something that works for me. Thank you again.
